templates/lib/yarp/serialize.rb.erb

```diff
@@ -110,10 +110,21 @@ module YARP
         end
       end
 
-      def load_string
+      def load_message
         io.read(load_varint).force_encoding(encoding)
       end
 
+      def load_string
+        case io.getbyte
+        when 1
+          input.byteslice(load_varint, load_varint).force_encoding(encoding)
+        when 2
+          load_message
```

Comment:
`load_message` feels confusing here because this might not be a "message string".
How about `load_owned_string` or `load_unshared_string`?
(same in Loader.java)

Reply:
If we want to use other terms, maybe `load_source_string` vs `load_serialized_string`

Reply:
I renamed it to `load_embedded_string` since that matches more of the Ruby codebase ideas.